src/HTTP.jl

```diff
@@ -1,16 +1,16 @@
 __precompile__(true)
 module HTTP
 
-export Request, Response, FIFOBuffer
-
 using MbedTLS
-
+import MbedTLS.SSLContext
 const TLS = MbedTLS
 
-import Base.==
 
-const DEBUG = false
-const PARSING_DEBUG = false
+import Base.== # FIXME rm
```

Comment:
why remove this?

Reply:
I think my intention was that the custom == methods should go in the submodule that defines the relevant custom type. e.g. that == should not be imported into the top-level HTTP module.